keadm/cmd/keadm/app/cmd/helm/cloudcore.go

```diff
@@ -66,6 +66,7 @@ const (
 )
 
 var setsKeyImageTags = []string{"cloudCore.image.tag", "iptablesManager.image.tag", "controllerManager.image.tag"}
+var setsKeyImageRepositories = map[string]string{"cloudCore.image.repository": "kubeedge/cloudcore", "iptablesManager.image.repository": "kubeedge/iptables-manager", "controllerManager.image.repository": "kubeedge/controller-manager"}
```

Comment:
This is the best flag imageRepository format: `{registry}/{space}` (e.g. my.harbor.cn/my-kubeedge), this allows more flexibility to customize the image.
```golang
var setsKeyImageRepositoriesAndNameMapper = map[string]string{
    "cloudCore.image.repository": "cloudcore", 
    "iptablesManager.image.repository": "iptables-manager", 
    "controllerManager.image.repository": "controller-manager"
}

for k, v := range setsKeyImageRepositories {
    sets = append(opts.Sets, fmt.Sprintf("%s=%s", k, opts.ImageRepository+"/"+v))
}

```

Reply:
ok